bundles/org.openhab.binding.freeathomesystem/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="freeathomesystem"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!--Unknown Type -->
+	<thing-type id="unknown" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Unknown</label>
+		<description>Unknown device for Free@Home Binding</description>
+	</thing-type>
```

Comment:
What is the point of an unknown device? does it do anything?

Reply:
The unknown device shall be used for further development to detect currently not implemented/unknown devices. This device detection is deactivated in the code. If it is activated generates a line in the discovery.
For standard user not visible.

Reply:
So why does it need to be a real thing type? If I understand it is only for logging, you can just log there is an unknown device without there being a 'fake' non used `thing`? I probably dont understand what your trying to do here, hence why I ask. Just seems strange to create a fake thing that has no channels and does nothing.

Reply:
During the development it was comfortable to visualise the devices directly in the openHAB web-frontend, instead read the long json files.
But you have completely right it is not necessary, it can be solved with logging the unknown devices.

Reply:
Unknown device is replaced by logging of these unknown devices